disco_aws_automation/resource_helper.py

```diff
@@ -37,6 +37,15 @@ def tag2dict(tags):
     return {tag.get('Key'): tag.get('Value') for tag in tags or {}}
 
 
+def dict2tag(dicts):
+    """
+    Converts the list of key:value strings (example ["mykey:myValue", ...])
+    into a list of AWS tag dicts (example: [{'Key': 'mykey', 'Value': 'myValue'}, ...]
+    """
+    return [{'Key': tag_key_value[0], 'Value': tag_key_value[1]}
+            for tag_key_value in [key_value_option.split(":") for key_value_option in dicts]]
```

Comment:
Per @aoskotsky-amplify, we are going to exclude `:` from the list of valid character used for Tag Key or value.